packaging/templates/mainnet-v1/sentry/sentry/bor/config.toml

```diff
@@ -4,7 +4,7 @@ chain = "mainnet"
 # vmdebug = false
 datadir = "/var/lib/bor/data"
 # ancient = ""
-# "db.engine" = "leveldb"
+# db.engine = "leveldb"
```

Comment:
Why were quotes removed here, all the flags with `.` should be wrapped within the quotes for proper parsing. I think this config file will fail if someone tries to uncomment the engine flag.

Reply:
Updated, this was tested prior, reverted to using the "" for the db engine, in default it uses the level db.